Bottom: db57b709633f31c5e3249fa80208c068ca54606e
Top:    d5bdc4719e6fa6d97cb112adce59a1b3d2a72c79
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-05 17:31:52 +0100

Refresh of support-holes-in-input-polygon

---

diff --git a/src/polygon.c b/src/polygon.c
index 2a95758..971481f 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -1884,7 +1884,7 @@ debug_polygon (PolygonType *p)
         fprintf(stderr, "\t%d: %d\n", i, p->HoleIndex[i]);
     }
   else
-    fprintf (stderr, "it hs no holes\n");
+    fprintf (stderr, "it has no holes\n");
   pa = p->Clipped;
   while (pa)
     {
